test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatHelpOutputTests.cs

```diff
@@ -82,5 +101,32 @@ public void MainInternal_OnParsingSuccess_DoesNotShowsHelp(string[] args)
             Assert.DoesNotContain("Usage", output);
             Assert.Equal(1, exitCode);
         }
+
+        [Fact]
+        public void MainInternal_ShowsHelp()
+        {
+            // Arrange
+            var originalConsoleOut = Console.Out;
+            using var consoleOutput = new StringWriter();
+            Console.SetOut(consoleOutput);
+            var log = new TestCommandOutputLogger(_testOutputHelper);
+
+            // Act
+            var exitCode = Program.MainInternal(Array.Empty<string>(), log);
+            Console.SetOut(originalConsoleOut);
+
+            // Assert
+            var output = consoleOutput.ToString();
+
+            var commandPattern = @"^\s{2}(\w+)\s{2,}"; // Matches lines starting with two spaces, a word (command), followed by at least two spaces
```

Comment:
Is this behavior always same? I mean, will each line have 2 spaces + command + 2 spaces?

Reply:
For now yes, I know this is not the best way to rely, when I finish the migration to the command I will make sure there is an easier way to know which commands are implemented.

Reply:
Is there an epic where you are tracking the work for migration? If yes, I suggest adding a tracking issue so that we don't forget.